resource_customizations/kubevirt.io/VirtualMachineInstance/health.lua

```diff
@@ -0,0 +1,15 @@
+hs = { status="Progressing", message="No status available"}
```

Comment:
For a VMI, there's a stable [`.status.phase`](https://github.com/kubevirt/client-go/blob/148fa0d1c7e83b7a56606a7ca92394ba6768c9ac/api/v1/types.go#L549) field which can be used (alongside the Ready and Paused conditions).
Here's a reasonable mapping to Argo CD health states:
- Pending, Scheduling, Scheduled --> Progressing
- Succeeded --> Suspended (can also use Healthy but I think Suspended conveys the more significant aspect here)
- Failed --> Degraded (can also use Suspended and have the `hs.message` discriminate between Succeeded/Failed/Paused)
- Unknown --> Unknown
- Running -->
    - Paused --> Suspended
    - !Ready --> Degraded
    - Ready --> Healthy